Jenkinsfile_CNP

```diff
@@ -12,8 +12,8 @@ def component = "user-profile-api"
 
 // Variables to switch pipeline logic and wiring per type of build
 def definitionStoreDevelopPr = "PR-575" // This doesn't change frequently, but when it does, only change this value.
-def userProfileApiDevelopPr = "PR-224" // This doesn't change frequently, but when it does, only change this value.
-def prsToUseAaat             = "PR-296" // Set this value to a PR number, or add it as a comma-separated value, if it's to follow CI/CD.
```

Comment:
Do you want to change this or do you want to make it comma separated like the below?
`def prsToUseAaat             = "PR-296, PR-306"`

Reply:
def prsToUseAaat = "PR-306" is better, since 296 was already merged.

Reply:
I wasn't aware of the status of 296 at the time of commenting. That makes sense 👍